torch/_dynamo/testing.py

```diff
@@ -252,7 +253,7 @@ def standard_test(self, fn, nargs, expected_ops=None, expected_ops_dynamic=None)
         expected = CompileCounter()
         try:
             gm = torch.fx.symbolic_trace(fn)
-            expected(gm)
+            expected(gm, [])
```

Comment:
I removed the param earlier, but I guess the right fix is to add the missing argument here

Reply:
Ended up using a type-igore/FIXME instead. Details: https://github.com/pytorch/pytorch/issues/112230